File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/shared/selectores/empleado/dominio/FiltroPaginacionCumpleanio.ts
import { ParametroPaginacionGenerico } from '../../../dominio/ParametroPaginacionGenerico';
export class FiltroPaginacionCumpleanio {
    constructor() {
        this.paginacion = new ParametroPaginacionGenerico();
    }
    paginacion: ParametroPaginacionGenerico;

    idPersona: number;
    idCentroCostos: string;
    fechaInicio: Date;
    fechaFin: Date;
    compania: string;
    dia: number;
    mes: number;
}

File: Administrativo/spring-erp-servicio/net/royal/spring/core/dominio/filtro/FiltroPaginacionCumpleanio.cs
using net.royal.spring.framework.core.dominio;
using System;
using System.Collections.Generic;
using System.Text;

namespace net.royal.spring.core.dominio.filtro
{
    public class FiltroPaginacionCumpleanio
    {


        public Int32? IdPersona { get; set; }
        public String IdCentroCostos { get; set; }
        public DateTime? FechaInicio { get; set; }
        public DateTime? FechaFin { get; set; }
        public string compania { get; set; }

        public ParametroPaginacionGenerico paginacion { get; set; }

        public FiltroPaginacionCumpleanio()
        {
            paginacion = new ParametroPaginacionGenerico();
        }
    }
}
